service/src/main/java/org/mskcc/cmo/metadb/service/impl/SampleServiceImpl.java

```diff
@@ -168,4 +176,10 @@ public Boolean sampleHasMetadataUpdates(SampleMetadata existingSampleMetadata,
         }
         return Boolean.FALSE;
     }
+
+    @Override
+    public PublishedMetadbSample getPublishedMetadbSample(UUID metadbSampleId) throws ParseException {
+        MetadbSample sample = sampleRepository.findSampleById(metadbSampleId);
```

Comment:
remember that the sample object returned here is not going to have all of its relationships actually filled in which is why we have to call a method specifically when calling `saveSampleMetadata(...)` in the sample service to fill in the missing details. 

So for example this `MetadbSample` returned by this method isn't going to have any of its sample aliases linked and the `metadbPatientId` and `metadbSampelId` values aren't going to be populated either.